owncloudApp/src/main/res/values/strings.xml

```diff
@@ -748,7 +748,7 @@
     <string name="release_notes_4_3_0_title_clear_data_button_hard_reset">"Clear data" button enabled in the app settings in device settings</string>
     <string name="release_notes_4_3_0_subtitle_clear_data_button_hard_reset">The "Clear data" button has been enabled to delete the application data from the app settings in the device settings. The application will perform a hard reset if this button is pressed</string>
     <string name="release_notes_4_3_0_title_improvements_remove_dialog">Improvements in remove dialog</string>
-    <string name="release_notes_4_3_0_subtitle_improvements_remove_dialog">Added a custom dialog when the file that is going to be deleted has thumbnail and added the number of files that are going to be deleted in a multiple selection</string>
+    <string name="release_notes_4_3_0_subtitle_improvements_remove_dialog">Modified the dialog when the file that is going to be deleted has thumbnail and added the number of files that are going to be deleted in a multiple selection</string>
```

Comment:
Same as in calens... Why modified? Users won't have seen the previous release note by the time they see this one, so we can not say that it has been modified because for them this is going to be brand new! Also, wasn't this PR about showing the dialog for EVERY file, not only for those that have thumbnail as the release note says? We're not giving correct information here

Reply:
Changed